packages/expo-doctor/jest.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('jest').Config} */
+module.exports = {
+  ...require('expo-module-scripts/jest-preset-cli.js'),
+  clearMocks: true,
+  displayName: require('./package').name,
+  rootDir: __dirname,
+  roots: ['__mocks__', 'src'],
+};
```

Comment:
Since we build with `ncc`, which doesn't use babel, we probably can switch Jest to use TS-Jest instead and get rid of the babel config. Also helps keeping 1 build system for both testing and publishing 😄 

```suggestion
module.exports = {
  ...require('expo-module-scripts/jest-preset-cli.js'),
  preset: 'ts-jest',
  clearMocks: true,
  displayName: require('./package').name,
  rootDir: __dirname,
  roots: ['__mocks__', 'src'],
};
```